src/firegraph/Collection.ts

```diff
@@ -10,37 +13,46 @@ import { setQueryFilters } from './Where';
  * @param store An initialized Firestore instance.
  * @param collectionName The path of the collection we want to retrieve.
  * @param selectionSet The rules for defining the documents we want to get.
+ * @param cacheManager An instance of cache manager to pass on to the documents fetcher
  */
 export async function resolveCollection(
-    store: firebase.firestore.Firestore,
-    collectionName: string,
-    collectionArgs: { [key:string]: any },
-    selectionSet: GraphQLSelectionSet
+  store: firebase.firestore.Firestore,
+  collectionName: string,
+  collectionArgs: { [key: string]: any },
+  selectionSet: GraphQLSelectionSet,
+  cacheManager: CacheManager
 ): Promise<FiregraphCollectionResult> {
-    let collectionQuery: any = store.collection(collectionName);
-    let collectionResult: FiregraphCollectionResult = {
-        name: collectionName,
-        docs: []
-    };
+  let collectionQuery: any = store.collection(collectionName);
+  let collectionResult: FiregraphCollectionResult = {
+    name: collectionName,
+    docs: [],
+  };
 
-    if (collectionArgs) {
-        if (collectionArgs['where']) {
-            const where = collectionArgs['where'];
-            collectionQuery = setQueryFilters(collectionQuery, where);
-        }
+  if (collectionArgs) {
+    if (collectionArgs['where']) {
+      const where = collectionArgs['where'];
+      collectionQuery = setQueryFilters(collectionQuery, where);
     }
 
-    const collectionSnapshot = await collectionQuery.get();
-    if (selectionSet && selectionSet.selections) {
-        for (let doc of collectionSnapshot.docs) {
-            const documentPath = `${collectionName}/${doc.id}`;
-            collectionResult.docs.push(await resolveDocument(
-                store,
-                documentPath,
-                selectionSet,
-                doc
-            ));
-        }
+    if (collectionArgs['limit']) {
+      const limit: number = Number.parseInt(collectionArgs['limit']);
+      collectionQuery = collectionQuery.limit(limit);
     }
-    return collectionResult;
-}
\ No newline at end of file
+
+    if (collectionArgs['orderby']) {
```

Comment:
This arg name should be `orderBy` to be consistent with Firestore naming.

Reply:
sure.